R/get_smr.R

```diff
@@ -1,96 +1,162 @@
-#' Perform indirect rate standardization
+#' Calculate a standardized morbidity/mortality ratio
 #'
-#' @description Compute a standardized mortality ratio (SMR) or expected number
-#' of deaths given the number of observed deaths, the size of each stratum in
-#' the observed population, and the specific rate for each stratum in the
-#' standard population.
+#' @description Calculate a standardized morbidity/mortality ratio (SMR) or the
+#' expected number of events in each stratum.
 #'
-#' @param count A scalar, the number of observed deaths.
-#' @param popn A numeric vector containing the size of each defined population.
-#' @param std_rt A numeric vector containing the specific rate (expressed as
-#' the number of cases per unit of population) for each stratum in the standard
+#' @param count A scalar, the number of observed events.
+#' @param popn A numeric vector containing the size of each stratum in the observed
 #' population.
-#' @param output_type If `"ratio"` (default), only the SMR is returned. If
-#' `"counts"`, the expected number of deaths for each stratum in the observed
-#' population is returned as a vector.
-#' @param percent If `FALSE` (default), the SMR is returned as is (e.g. 0.5).
-#' If `TRUE`, the SMR is expressed as a percentage (e.g. 50).
+#' @param std_r A numeric vector containing the rate or risk of each stratum in
+#' the standard population.
+#' @param measure A string to indicate which type of SMR to calculate. Currently,
```

Comment:
I think we should describe it a little bit differently. The choice between `rate` and `risk` is not really a "type" of SMR---either way it is still the SMR which is always observed events / expected events. The choice is the measure used to calculate the expected events (either using rate or risk in the standard population) and this is linked to `std_r`: the function needs to know what is being passed in.

I would suggest changing this parameter to `std_measure` to match `std_r` and the description should be:
>A string to indicate whether the values in `std_r` are rates or risks, which is used to calculate the expected number of events in each stratum. Currently ... [continue the rest as before]